.github/workflows/ci.yml

```diff
@@ -10,6 +10,59 @@ env:
   CARGO_TERM_COLOR: always
 
 jobs:
+  hub:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Prepare
+        run: |
+          echo "BAKE_TARGET=image-cross" >> $GITHUB_ENV
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.REPO_SLUG }}
+          ### frontend versioning
+          ### on semver tag:
+          # second-state/runwasi:1.2.3
+          # second-state/runwasi:1.2
+          # second-state/runwasi:1
+          # second-state/runwasi:latest
+          ### on pre-release tag:
+          # second-state/runwasi:1.1.0-rc.1
+          ### on push default branch (main):
+          # second-state/runwasi:main
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
```

Comment:
will not have any effect as `semver` is only set on push tag event. Would need to set:

```yaml
on:
  push:
    branches:
      - 'main'
    tags:
      - 'v*'
  pull_request:
```

Reply:
Removed since there is no tagging for now